tests/data/basic_subset.tsv

```diff
@@ -13,6 +13,7 @@
 #  b: "http://example.org/b/"
 #  c: "http://example.org/c/"
 #  d: "http://example.org/d/"
+#  orcid: "https://orcid.org/"
```

Comment:
Why did we only now realise this was missing? Weird.

Reply:
Yeah weird. Because this test tsv never went through `clean_prefixes()` until this time.